public/app/features/alerting/unified/components/rules/RuleDetailsMatchingInstances.tsx

```diff
@@ -79,7 +79,7 @@ function filterAlerts(
   }
   if (alertInstanceState) {
     filteredAlerts = filteredAlerts.filter((alert) => {
-      return alert.state === alertInstanceState;
+      return alert.state.split(' ')[0] === alertInstanceState;
```

Comment:
I think a function like `matchMappedAlertStateToInitialAlertState()` would make more sense instead of `split(' ')[0]`, would you mind adding that one?

Reply:
Sure, happy to encapsulate this. Are you fine with an in-line named function or do you think it should exist at the top-level of the component like `filterAlerts` does, or at the module level?